### DISCUSSION TITLE
Problem with One Sample Test in Jamovi

### DISCUSSION THREAD

**Discussion by lprimvera** [community/community#140125] [New to GitHub]
If you do a one sample t-test in Jamovi there is a choice to put in the Population mean choose a non-directional or directional hypothesis.  For the alternate hypothesis Mu > M, the p value comes out 1.0000.  The t-test value for tests are correct as are the p values.  Not for this one.

**Comment by BhaktiMore18** [comment]
In Jamovi, when running a one-sample t-test with the alternative hypothesis "Mu > M" (i.e., testing if the sample mean is greater than the population mean), a p-value of 1.0000 indicates that the sample mean is less than or equal to the population mean, providing no evidence to support the alternative hypothesis.

If this result seems unexpected, double-check that your hypothesis direction matches your expectations. For more general comparisons, consider using a two-tailed (non-directional) hypothesis. It may also be helpful to review your data and ensure the population mean is correctly entered.

**Comment by lprimvera** [comment]
Thank you.  This is very helpful.

Lou Primavera


[https://touro.box.com/shared/static/0eot2o1gmk4yow2hl4bwyvifyabvj130.png]
Louis H. Primavera Ph.D.
Associate Provost
Office of the Provost
Cell: 516.662.3454
***@***.******@***.***>
www.touro.edu<http://touro.edu/>
[facebook]<https://urldefense.proofpoint.com/v2/url?u=http-3A__facebook.com_wearetouro&d=DQMFAg&c=odzYs1kPF7h99M0Vn1uLzg&r=AbAFlD9fKKb1JUz09TFcfsS2csApBvg2TpwKtVPD0h0&m=xOkCtlsE-QYS3rHeVa0W7KsZNBwaNzH3Oh8gd-3_ApA&s=KD3t6uUrTN_GR9rZO_Cep-A2y4DOJBje8uln9IkC9kI&e=>
[https://touro.box.com/shared/static/dfvz431yevi05t92ahms9fum8vxk27sn.png]<https://urldefense.proofpoint.com/v2/url?u=http-3A__twitter.com_wearetouro&d=DQMFAg&c=odzYs1kPF7h99M0Vn1uLzg&r=AbAFlD9fKKb1JUz09TFcfsS2csApBvg2TpwKtVPD0h0&m=xOkCtlsE-QYS3rHeVa0W7KsZNBwaNzH3Oh8gd-3_ApA&s=pIqW9bjoHIXkC5CcX48hfqwehT-MpR27FpYEzOdYLdg&e=>
[https://touro.box.com/shared/static/8t6hz22v2g61frfkkh009o5rpdzn9uw4.png]<https://urldefense.proofpoint.com/v2/url?u=http-3A__instagram.com_wearetouro&d=DQMFAg&c=odzYs1kPF7h99M0Vn1uLzg&r=AbAFlD9fKKb1JUz09TFcfsS2csApBvg2TpwKtVPD0h0&m=xOkCtlsE-QYS3rHeVa0W7KsZNBwaNzH3Oh8gd-3_ApA&s=6sSXOkaSRAlBr14ITcJ_wacRiL1BbRX9zy_XAeznCd8&e=>
[https://touro.box.com/shared/static/ujo3knaz4n715fw34bc3ksh5w3u1odhd.png]<https://urldefense.proofpoint.com/v2/url?u=https-3A__www.linkedin.com_edu_school-3Fid-3D19065&d=DQMFAg&c=odzYs1kPF7h99M0Vn1uLzg&r=AbAFlD9fKKb1JUz09TFcfsS2csApBvg2TpwKtVPD0h0&m=xOkCtlsE-QYS3rHeVa0W7KsZNBwaNzH3Oh8gd-3_ApA&s=PoSuicblXoOHXIhjLrudoOyrHe6-j_7Y6olKcFC8PBk&e=>
